test/integration/full/controls/project-factory.rb

```diff
@@ -51,10 +63,6 @@
       end
     end
 
-    it "includes the Google App Engine API service account user" do
```

Comment:
@thefirstofthe300 Please do not introduce completely unrelated changes into a different PR. Is this somehow related to the other functionality in this PR and I missed it?